index.html

```diff
@@ -1,15 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <script src="script.js"></script>
 </head>
+
 <body>
-    <h1>this is my first line of code </h1>
+    <h1>this is the new heading </h1>
```

Comment:
Thanks for the this change this was very helpfull

Reply:
thanks ashu for this review

Reply:
Can you do more contribution to this repo

Reply:
yeah offcourse im going to do this for me